test/test.try.js

```diff
@@ -6,63 +6,114 @@ casper.options.waitTimeout = 20000;
 casper.start();
 
 casper.thenBypassUnless(function() {
+  console.log(apiKey);
```

Comment:
I killed the travis build before it got this far (which would have made it public) and removed the line in https://github.com/watson-developer-cloud/visual-recognition-nodejs/commit/bd586591414ab5eb3b11159b0ff43f904ed61ee2